modules/sdk-core/src/bitgo/enterprise/iEnterprise.ts

```diff
@@ -12,4 +15,11 @@ export interface IEnterprise {
   getFirstPendingTransaction(params?: Record<string, never>): Promise<any>;
   settlements(): ISettlements;
   affirmations(): IAffirmations;
+  getAndVerifyEcdsaBitGoChallengeProofs(userPassword: string): Promise<BitGoProofSignatures>;
+  uploadAndEnableEcdsaSigning(
```

Comment:
enabling signing is a side effect of this fn, but what it actually does is just upload their challenge.

```suggestion
  uploadTssEcdsaChallenge(
```